client/src/components/NewColumnDialogBox/NewColumnDialogBox.tsx

```diff
@@ -54,15 +54,9 @@ const NewColumnDialogBox: FC<ColumnDialogFormProps> = ({ open, handleClose, deta
                 closeBackdrop();
               }, 1200);
               if (details.position === 'left') {
-                updateBoard([
-                  { cards: [{ ...details.draggedCard }], id: values.title, title: values.title },
-                  ...details.boardArrangement,
-                ]);
+                updateBoard([{ cards: [{ ...details.draggedCard }], name: values.name }, ...details.boardArrangement]);
               } else {
-                updateBoard([
-                  ...details.boardArrangement,
-                  { cards: [{ ...details.draggedCard }], id: values.title, title: values.title },
-                ]);
+                updateBoard([...details.boardArrangement, { cards: [{ ...details.draggedCard }], name: values.name }]);
```

Comment:
@ajaytomgeorge, answering your other question, it's because of this part of the code. I will create a new card at this time, so I don't have an _id. If I do mandatory, the function will not allow me to update the board.